uisdk/src/main/java/com/karhoo/uisdk/screen/rideplanning/RidePlanningStorage.kt

```diff
@@ -0,0 +1,24 @@
+package com.karhoo.uisdk.screen.rideplanning
+
+import com.karhoo.sdk.api.model.Quote
+import com.karhoo.sdk.api.model.TripInfo
+import com.karhoo.sdk.api.network.request.PassengerDetails
+import com.karhoo.uisdk.screen.booking.checkout.loyalty.LoyaltyInfo
+import com.karhoo.uisdk.screen.booking.domain.address.JourneyDetailsStateViewModel
+import com.karhoo.uisdk.screen.booking.domain.address.JourneyInfo
+import com.karhoo.uisdk.screen.booking.domain.bookingrequest.BookingRequestStateViewModel
+
+object RidePlanningStorage {
```

Comment:
Since this stores more info that needed just for the ride planning screen, should we rename this to `BookingStorage`? It would better reflect the context in which we'll use this, which is the entire booking flow.